### PR TITLE
Move Icon Badges add on to Feature Flag

### PR DESCRIPTION
**Pull request by mkangia**
https://manage.dimagi.com/default.asp?265043#BugEvent.1419504

This moves the application add on Icon Badges to a feature flag. This would then be available only for enikshay. @sheelio is going to handle comms to other users if any using it.
Also adds command to find applications/domains using a particular add on or with all add ons enabled

### CODE REVIEW

File: corehq/apps/app_manager/management/commands/applications_with_add_ons.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                for application in domain.full_applications(include_builds=False):
+                    if not application.is_remote_app():
+                        all_add_ons_enabled = toggles.ENABLE_ALL_ADD_ONS.enabled(domain.name)
+                        if add_on_name in application.add_ons or all_add_ons_enabled:
+                            writer.writerow({
+                                'domain': domain.name,
+                                'application_id': application.get_id,
+                                'app_name': application.name,
+                                'all_add_ons_enabled': all_add_ons_enabled,
+                                'status': application.add_ons.get('custom_icon_badges')
```
Comment (orangejenny) [inline corehq/apps/app_manager/management/commands/applications_with_add_ons.py]: should be `add_on_name`

**Review by orangejenny — CHANGES_REQUESTED** [review]

File: corehq/apps/app_manager/management/commands/applications_with_add_ons.py (hunk 1 above)
Comment (mkangia) [reply corehq/apps/app_manager/management/commands/applications_with_add_ons.py]: oops, thanks for catching.

**Review by orangejenny — APPROVED** [review]
Updates look good, feel free to self-merge once py3 check passes.

**Comment by mkangia** [comment]
@sheelio unsure of what label this should have. 
`product/feature-flag` + `product/all-users-all-environment`?

**Comment by sheelio** [comment]
I would use all users-all-environments (since we're changing behaviour for potentially non feature flagged projects).

**Pull request merged**